packages/jss-plugin-rule-value-observable/src/index.js

```diff
@@ -10,10 +10,34 @@ import {
 } from 'jss'
 import type {Observable} from './types'
 
-const isObservable = value => value && value[$$observable] && value === value[$$observable]()
+const isObservable = (value: any) => value && value[$$observable] && value === value[$$observable]()
 
 export type Options = UpdateOptions
 
+function getObservableStyles(styles: Object): Object | null {
```

Comment:
We will need it for proper integration into `react-jss`, but I will make that in a different PR.

Reply:
What integration are you talking about? mb we should do it differently in that case.

Reply:
Proper `observables` integration into react-jss

Reply:
But I'm also not sure If we actually need `observable` support in `react-jss`.

Reply:
Yes, but what exactly do you mean? Ability to have a new observable per element?

Reply:
No, right now you need to pass `link: true` manually to the sheet options. And we also create a dynamic sheet for observables even though there is no need for it.

Reply:
Ah this, yeah, mb lets fix this in the same step, because depending on this we might want to have   a different utility function and its location

Reply:
I don't think the fix should go into this PR though.

Reply:
You mean because those are different topics? This PR is not of a big practical nature anyways. We can mix those things in that PR I think. Its not too big either and tightly related.

Reply:
If we end up having a function that knows about both, functions and observables, it shouldn't be in one of those plugins, but somewhere else. Duplicating that function in both is also not very nice, it might need some adjustments in the future and having it twice doesn't make it easier to maintain.

Reply:
But why do we need to have a function which knows about both?

Reply:
I agree that core is not the ideal place for them, but I want to keep performance-first constraint. If we need to loop over styles twice just for the sake of functions living in separate packages, it is not quite performance-first.

Reply:
The only place we are using `getDynamicStyles` at the moment is in `react-jss`. And without #1002 there will be no performance loose.